test/parallel/test-console.js

```diff
@@ -105,8 +103,10 @@ console.timeEnd('constructor');
 console.time('hasOwnProperty');
 console.timeEnd('hasOwnProperty');
 
-global.process.stdout.write = stdout_write;
-global.process.stderr.write = stderr_write;
+assert.strictEqual(strings.length, process.stdout.writeTimes);
```

Comment:
In fact, `listener` is sync which means a `write` action is emitted but not finished.

I just added the `callback` to the original `_write` function to make it may receive`callback`.